cloudant-sync-datastore-core/src/test/java/com/cloudant/sync/datastore/DatabaseManagerTest.java

```diff
@@ -47,11 +44,11 @@ public void tearDown() {
         FileUtils.deleteQuietly(TEST_PATH);
     }
 
-    @Test(expected = IllegalArgumentException.class)
-    public void nonWritableDatastoreManagerPathThrows() throws DatastoreNotCreatedException {
+    @Test(expected = DatastoreNotOpenedException.class)
+    public void nonWritableDatastoreManagerPathThrows() throws DatastoreNotOpenedException, IOException {
         File f = new File(TEST_PATH, "c_root_test");
         try {
-            f.mkdir();
+            FileUtils.forceMkdir(f);
```

Comment:
"Makes a directory, including any necessary but nonexistent parent directories" (https://commons.apache.org/proper/commons-io/javadocs/api-2.5/org/apache/commons/io/FileUtils.html#forceMkdir(java.io.File)) seems like the right thing, no? This is necessary for #294.